views/scss/qtiCreator/_choice.scss

```diff
@@ -8,13 +8,22 @@
         .label-content {
             display: block;
         }
-
         .mini-tlb {
             @include rtl {
                 right: auto;
                 left: -38px;
             }
         }
+        .widget-math.edit-active{
+            span.mini-tlb{
+            @include rtl {
+                margin-right: 6.3em;
+                left: -2.20em;
+                right: auto;
+                 }
```

Comment:
Did you try to add some math numbers on the Latex input?
![image](https://user-images.githubusercontent.com/11692751/159668746-79c34130-ff8b-4f6c-a6d9-f87c1f064e31.png)

Reply:
Tip: the `right: auto` rule is not being applied anyway because there is an `!important` below that has priority
![image](https://user-images.githubusercontent.com/11692751/159669252-321843c2-6d8c-445c-98b1-9b35bfcb186a.png)